.github/actions/verify-token/action.yml

```diff
@@ -5,24 +5,25 @@ description: "Verify a token"
 inputs:
   slsa-workflow-recipient:
     description: "The audience of the token"
-    type: string
     required: true
 
   slsa-unverified-token:
     description: "The token to verify"
-    type: string
     required: true
 
   token:
     description: "The GitHub Actions token."
     required: false
-    default: ${{ github.token }}
```

Comment:
qq: why is the default here removed? the code doesn't look for GITHUB_TOKEN as a fallback (but maybe there's some magic i don't know about)

Reply:
Nope. This was a mistake. That's what we have reviews for I guess 😞